README.md

```diff
@@ -99,6 +105,28 @@ spec:
   - sensu/sensu-influxdb-handler
 ```
 
+### Handler definition for [Check Status]
+
+```yml
+---
+api_version: core/v2
+type: Handler
+metadata:
+  namespace: default
+  name: influxdb-check-status
+spec:
+  type: pipe
+  command: sensu-influxdb-handler -d sensu -c
+  timeout: 10
+  env_vars:
+  - INFLUXDB_ADDR=http://influxdb.default.svc.cluster.local:8086
+  - INFLUXDB_USER=sensu
+  - INFLUXDB_PASS=password
+  filters: []
+  runtime_assets:
+  - sensu/sensu-influxdb-handler
+```
+
```

Comment:
I appreciate the addition of an example to the readme, but this example seems a little redundant. Since the functionality of `-c` is already documented in the help usage, we can probably forego this example.

Reply:
Yeah, agree. I will take out this example.